jbpm-flow-builder/src/test/java/org/jbpm/process/workitem/WorkDefinitionImplTest.java

```diff
@@ -122,7 +122,7 @@ public void testParameterValuesWithEnumsOnly() throws Exception {
             if (entry.getKey().equals("param1")) {
                 EnumDataType paramEnum = (EnumDataType) entry.getValue();
                 assertEquals("org.jbpm.process.workitem.enums.AnimalsEnum", paramEnum.getClassName());
-                Map<String, Object> paramValuesMap = paramEnum.getValueMap();
+                Map<String, Object> paramValuesMap = paramEnum.getValueMap(null);
```

Comment:
@psiroky the method takes a ClassLoader as argument which however is optional. The case of a null arg is correctly managed inside it.